src/nub.ts

```diff
@@ -223,44 +223,14 @@ export class NUB {
     return transaction;
   }
 
-  public async approve(tokenAddress: string, amount: string) {
```

Comment:
@nonseodion what about the people using these methods?

Reply:
Since we are the only ones using it for now I reached out to Joseph if he was still using those methods he said he wasn't though. I still can't say for sure if he still uses them.

Reply:
I did a check on the frontend he still uses one or two of those functions I'll revert back

Reply:
Just fixed it in latest commit